app/js/controllers/actions/incomes/incomes-list-ctrl.js

```diff
@@ -22,6 +22,8 @@ angular
 		}
 
 		_.each($scope.data, element => {
+			element = preProcess.changeDate(element);
+			element.attributes.recievedOn = preProcess.convertTimeToEST(element.attributes.recievedOn);
```

Comment:
why is this necessary / not covered in ` preProcess.changeDate(element)`?

Reply:
changeDate() is a helper function that called convertTimeToEST() except it only calls it on the element's create and modified attributes. Since some pages have other fields that require the conversion, you can use convertTimeToEST() on them in their controller.